docs/static/ls-ls-config.asciidoc

```diff
@@ -18,7 +18,7 @@ Logstash-to-Logstash communication can be achieved in one of two ways:
 This is the preferred method to implement Logstash-to-Logstash. It replaces <<ls-to-ls-http>> and has these considerations:
 
 * It relies on HTTP as the communication protocol between the Input and Output.
-* It does not provide built-in high availability. You will need to implement your own load balancer in between the Logstash output and the Logstash input.
+* It provides built-in _fairness_ load balancing, which, distributes batch of events to a downstream that is running, has a capacity to receive events and no recent failures.
```

Comment:
this is a bit unclear:

What does it mean "has a capacity to receive events" ? How does it know?
What about the "no recent failures" ? does this mean it is not based on that?

Reply:
Also, I think we can simply summarize that we support loadbalancing when providing multiple outputs. And the explanation of how we distribute the load should be present in the other page, as per below comment.